docs/usage/verticles/ServerVerticle.md

```diff
@@ -0,0 +1,183 @@
+# ServerVerticle
+
+The `ServerVerticle` is a Vert.x verticle that exposes endpoints using the HTTP(S) protocol. NeonBee deploys the
+`ServerVerticle` automatically if the `WEB` [profile](./../neonbee.md#neonbee-profiles) is active. When the
+`ServerVerticle` is started, it creates a router and mounts the specified endpoints onto it. It then creates an HTTP
+server using the router. When the `ServerVerticle` is stopped, the HTTP server is closed and additional logging is
```

Comment:
why do you mention the additional logging here?

Reply:
removed